torch/_inductor/graph.py

```diff
@@ -1244,27 +1268,37 @@ def debug(msg: str) -> None:
         operation_watermark = len(self.operations)
 
         origins = {n}
-        if n.op == "call_function":
+        is_call_function = n.op == "call_function"
+        if is_call_function:
             args, kwargs = self.fetch_args_kwargs_from_env(n)
             origins |= gather_origins(args, kwargs)
         with ir.IRNode.current_origins(origins), self.set_current_node(  # type: ignore[arg-type]
             n
         ), V.set_current_node(
             n
         ):
+            if is_call_function:
+                self.maybe_add_custom_op_layout_constraints(n.target)  # type: ignore[arg-type]
```

Comment:
nit: could just return in this function to avoid the control flow. it already has `maybe` in its function name. it errors on some inputs but doesnt do anything on others..

Reply:
sounds good